components/StatusBase.vue

```diff
@@ -0,0 +1,15 @@
+<template>
```

Comment:
You could merge Status*.vue into one component.
Only formatter and URL are different, use props to pass them.